sass/themes/charts/_data-chart-theme.scss

```diff
@@ -50,7 +68,20 @@
     $marker-outlines: null,
     $plot-area-background: null,
     $title-horizontal-alignment: null,
-    $subtitle-horizontal-alignment: null
+    $subtitle-horizontal-alignment: null,
+    $axis-label-margin: null,
+    $axis-label-text-color: null,
+    $axis-label-text-style: null,
+    $axis-label-vertical-alignment: null,
+    $axis-major-stroke-thickness: null,
+    $axis-minor-stroke-thickness: null,
+    $axis-stroke: null,
+    $axis-stroke-thickness: null,
+    $axis-tick-length: null,
+    $axis-tick-stroke-thickness: null,
+    $axis-title-alignment: null,
+    $axis-title-margin: null,
+    $axis-title-text-style: null
```

Comment:
You have missed some of the chart properties here: 
$axis-title-text-color
$axis-major-stroke
$axis-minor-stroke
$axis-minor-stroke
$axis-tick-stroke